### PR TITLE
Add support for building CentOS Stream 8

### PR DESCRIPTION
**Pull request by phracek**
Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

**Comment by phracek** [comment]
@zmiklank @hhorak @pkubatrh  I would be glad for review and feedback

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
[test]

**Comment by phracek** [comment]
[test]

File: tag.sh [hunk 1]
```diff
@@ -19,11 +19,9 @@ for dir in ${VERSIONS}; do
   date_and_hash="${commit_date}-$(git rev-parse --short HEAD)"
 
   full_reg_name="$REGISTRY$name"
-  echo "-> Tagging image '$IMAGE_ID' as '$full_reg_name:$version' and '$full_reg_name:latest' and '$full_reg_name:$date_and_hash'"
-  if [[ $OS == "c9s" ]] || [[ $OS == "fedora" ]]; then
-    echo "-> Tagging image '$IMAGE_ID' as '$full_reg_name:$OS'"
-    docker tag "$IMAGE_ID" "$full_reg_name:$OS"
-  fi
+  echo "-> Tagging image '$IMAGE_ID' as '$full_reg_name:$version' and '$full_reg_name:latest' and '$full_reg_name:$OS' and '$full_reg_name:$date_and_hash'"
```
Comment (zmiklank) [inline tag.sh:22]: Could you please explain this change?
I would like to double-check because it affects also other distributions.

Comment (phracek) [reply tag.sh:22]: It does not make sense to tag image as `:$OS` only for c9s, c8s, and fedora. Let's do it for all images. But only c8s, c9s and fedora use it.

**Pull request merged**